azure-pipelines.yml

```diff
@@ -24,78 +24,68 @@ jobs:
   - script: |
       # XXX next command should avoid redefining the path in every step, but
       # made the process crash as it couldn't find deactivate
-      #echo '##vso[task.prependpath]$HOME/miniconda3/bin'
+      echo '##vso[task.prependpath]$(HOME)/miniconda3/bin'
       echo '##vso[task.setvariable variable=ENV_FILE]environment.yml'
       echo '##vso[task.setvariable variable=AZURE]true'
     displayName: 'Setting environment variables'
 
   # Do not require a conda environment
-  - script: |
-      export PATH=$HOME/miniconda3/bin:$PATH
-      ci/code_checks.sh patterns
+  - script: ci/code_checks.sh patterns
     displayName: 'Looking for unwanted patterns'
     condition: true
 
   - script: |
-      export PATH=$HOME/miniconda3/bin:$PATH
       sudo apt-get install -y libc6-dev-i386
       ci/setup_env.sh
+      echo "$PATH"
```

Comment:
do we want this, or was it for debugging? I guess was for debugging, if that's the case, can you please remove it